src/components/ha-selector/ha-selector-datetime.ts

```diff
@@ -72,7 +72,7 @@ export class HaDateTimeSelector extends LitElement {
 
   private _sendChangedEvent(): void {
     fireEvent(this, "value-changed", {
-      value: `${this._dateInput.value} ${this._timeInput.value}`,
+      value: `${this._dateInput.value ?? ""} ${this._timeInput.value}`,
```

Comment:
Should we not just only send a change event when we can send a valid value? So when we have both a date and time?